modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl

```diff
@@ -2926,6 +2952,19 @@ public class ${entity.name}PersistenceImpl extends BasePersistenceImpl<${entity.
 		</#list>
 	}
 
+	<#if !serviceBuilder.isVersionGTE_7_1_0()>
+		@Override
+		public void setSessionFactory(SessionFactory sessionFactory) {
+			super.setSessionFactory(sessionFactory);
+
+			DBType dbType = DBManagerUtil.getDBType(sessionFactory.getDialect());
+
+			_databaseInMaxParameters = GetterUtil.getInteger(
+				PropsUtil.get(
+					"database.in.max.parameters",
+					new Filter(dbType.getName())));
+	</#if>
+
```

Comment:
@marianoalvarosaiz, 
1. `setSessionFactory() ` lacks of `}`. 
2. SessionFactory class doesn't been imported. You may firstly apply the change to one EntityNamePersistenceImpl class to check if it can be compiled passed.
3. SF, please use `elseIf` for `setSessionFactory()`, like the below
```
<#if dependencyInjectorDS>
		<#include "persistence_references.ftl">

		<#if serviceBuilder.isVersionLTE_7_2_0()>
			private boolean _columnBitmaskEnabled;
		</#if>
</#if>
<#elseif !serviceBuilder.isVersionGTE_7_1_0()>
		public void setSessionFactory(SessionFactory sessionFactory) {
}
</#if>
```